api/src/client/ApiTaxClearanceCertificateClient.test.ts

```diff
@@ -412,6 +412,69 @@ describe("TaxClearanceCertificateClient", () => {
     });
   });
 
+  describe("unlink tax ID", () => {
+    const originalEnvironment = process.env.STAGE;
+
+    afterEach(() => {
+      process.env.STAGE = originalEnvironment;
+    });
+
+    it("works in non-prod env", async () => {
```

Comment:
pebble - can the test name be more descriptive... i.e. it returns a successful response when...